tools/istio-docker.mk

```diff
@@ -27,6 +28,14 @@ DOCKER_TARGETS ?= docker.pilot docker.proxytproxy docker.proxyv2 docker.app dock
 	docker.mixer docker.mixer_codegen docker.citadel docker.galley docker.sidecar_injector docker.kubectl docker.node-agent-k8s \
 	docker.istioctl
 
+ifeq ($(TARGET_ARCH),amd64)
+	DOCKER_OPT=--build-arg DISTROLESS_IMAGE=gcr.io/distroless
+else ifeq ($(TARGET_ARCH),arm64)
+	DOCKER_OPT=--build-arg DISTROLESS_IMAGE=discolix
```

Comment:
DISTROLESS_IMAGE=`docker.io/discolix`. This is necessary as the CRI in use may not always use docker as a default. This project always encodes the HUB in every image.